src/controllers/midi/midimessage.h

```diff
@@ -204,13 +212,32 @@ struct MidiInputMapping {
     }
 
     bool operator==(const MidiInputMapping& other) const {
-        return key == other.key && options == other.options &&
-                control == other.control && description == other.description;
+        return std::visit([&](auto first, auto second) {
+            using T = std::decay_t<decltype(first)>;
+            using U = std::decay_t<decltype(second)>;
+
+            if constexpr (!std::is_same_v<T, U>) {
+                return false;
+            } else if constexpr (std::is_same_v<T, ConfigKey>) {
+                return key == other.key &&
+                        options == other.options &&
+                        std::get<ConfigKey>(control) == std::get<ConfigKey>(other.control) &&
+                        description == other.description;
+            } else if constexpr (std::is_same_v<T, QJSValue>) {
+                return key == other.key && options == other.options &&
+                        std::get<QJSValue>(control).strictlyEquals(
+                                std::get<QJSValue>(other.control)) &&
+                        description == other.description;
```

Comment:
Does this prevent from inserting the same lamdba multiple times?

Reply:
I think so. If it refers to the same lambda (obviously it won't work if different lambdas with the same body are declared, but that's fine IMO).
```javascript
> const a = () => {}
> const b = () => {}
> const c = a
> a === a
true
> a === b
false
> c === a
true
```